src/Microsoft.Health.Dicom.Api/Registration/DicomServerServiceCollectionExtensions.cs

```diff
@@ -146,6 +148,8 @@ public static IDicomServerBuilder AddHostedServices(this IDicomServerBuilder ser
 
         services.TryAddSingleton<RecyclableMemoryStreamManager>();
 
+        services.AddSingleton<ITelemetryInitializer, TelemetryInitializer>();
```

Comment:
Perhaps move this up next to the `AddApiVersioning` call

Reply:
I am looking at it as a generic telemetry extension which happens to do APIVersion column now.